### ISSUE TITLE
maybe useful idea

### ISSUE THREAD

**Issue by sewim** [alreadydone/lz#84]
First of all, It might be rediculous idea.
As you already knew AI seems to be stepped back when they are winning enough.
I saw interesting manual method for that issue.
Some players who is using lizzie make black and white opposite therefore at least their winrate successfully drop down and solve that issue.
Maybe if you can make it as a cord which works real time, There is possiablity that AI will be increased performence.
Thanks.

**Comment by alreadydone** [comment]
I think you can just use my komi+next branch (or the release of the same name) and append --nonslack to your command. It will automatically adjust winrate to the range 10-90% and you can also specify --max-wr and --min-wr if that's not enough. Latest 15b nets works well, but 40b nets may not.

**Comment by sewim** [comment]
Yes I had used your piece of art. It was good but It didn't work well for elf and 40b.
I think Maybe...just maybe converting B&W can be applied for any weights comparing with color input plane method.

**Comment by alreadydone** [comment]
Setting --target-komi -7.5 has absolutely the same effect as switching black and white. I get other komi by interpolating or extrapolating from these two scenarios.
